### PR TITLE
Change text definition intratelencepalic-projecting glutamatergic neurons in CL

### LINKED ISSUE #2339: Change text definitionintratelencepalic-projecting glutamatergic neurons in CL. Consider including their morphological and transcriptomic characteristics and just insert the layer of origin in the "comment" section
Change text definition for intratelencepalic-projecting glutamatergic neurons in CL. Mention transcriptomic distinction, include morphological and electrophysiological definition where possible, mention soma location where possible. Add standard transcriptomic reference data from CellxGene census.

### PR DESCRIPTION
**Pull request by AvolaAmg**
Fixes #2339

Changes made:

**intratelencephalic-projecting glutamatergic cortical neuron**
	Change reference in text def
	add reference in synonym
	Change synonym
	Change symbol

**L2/3-6 intratelencephalic projecting glutamatergic neuron**
	Add CxG link

**L2 intratelencephalic projecting glutamatergic neuron**
	L2 IT

**L2/3 intratelencephalic projecting glutamatergic neuron**
	Change text def
		Add database cross reference
	Add comments
	Add database cross reference
	Add symbol

**L4 intratelencephalic projecting glutamatergic neuron**
	Change text def
		Add database cross reference
	Add comments
	Add symbol

**L5 intratelencephalic projecting glutamatergic neuron**
	Change text def
		Add database cross reference
	Add symbol
	Add database cross reference

**L6 intratelencephalic projecting glutamatergic neuron**
	Change text def
		Add database cross reference
	Add symbol
	Add database cross reference

**L6 intratelencephalic projecting Car3 glutamatergic neuron**
	Change text def
		Add database cross reference
	Add symbol

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31134,27 +31140,33 @@ EquivalentClasses(obo:CL_4030062 ObjectIntersectionOf(obo:CL_4023040 ObjectSomeV
...
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4030063 <https://orcid.org/0000-0003-4389-9821>)
 AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030063 "2023-10-10T14:11:25Z"^^xsd:dateTime)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292694") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4030063 "L4 IT")
+AnnotationAssertion(rdfs:comment obo:CL_4030063 "Historically, the L4 intratelencephalic projectic glutametergic neuron was known to have its soma in cortical layer 4. MERFISH data showed that this intratelencephalic-projecting glutametergic neuron can have its also soma in layer 3.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:31148]: Grammatical mistake: 
```suggestion
AnnotationAssertion(rdfs:comment obo:CL_4030063 "Historically, the L4 intratelencephalic projectic glutametergic neuron was known to have its soma in cortical layer 4. MERFISH data showed that this intratelencephalic-projecting glutametergic neuron can also have its soma in layer 3.")
```

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -31134,27 +31140,33 @@ EquivalentClasses(obo:CL_4030062 ObjectIntersectionOf(obo:CL_4023040 ObjectSomeV
...
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292694") Annotation(oboInOwl:hasDbXref "PMID:37609206") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4030065 "An intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 6.")
+AnnotationAssertion(obo:IAO_0000028 obo:CL_4030065 "L6 IT")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292694") Annotation(oboInOwl:hasDbXref "PMID:37609206") Annotation(oboInOwl:hasDbXref "PMID:37824655") Annotation(oboInOwl:hasDbXref "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/") obo:IAO_0000115 obo:CL_4030065 "An transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in L6 of the primary motor cortex. These cells are short untufted pyramidal cells, which could be stellate or inverted. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Phenotypic variation of transcriptomic cell types in mouse motor cortex', dataset: 'Excitatory neurons integrated with 10X sequencing MOp data', Author Categories: 'BICCN_subclass_label', L6 IT.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:31166]:
```suggestion
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292694") Annotation(oboInOwl:hasDbXref "PMID:37609206") Annotation(oboInOwl:hasDbXref "PMID:37824655") Annotation(oboInOwl:hasDbXref "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/") obo:IAO_0000115 obo:CL_4030065 "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in L6 of the primary motor cortex. These cells are short untufted pyramidal cells, which could be stellate or inverted. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Phenotypic variation of transcriptomic cell types in mouse motor cortex', dataset: 'Excitatory neurons integrated with 10X sequencing MOp data', Author Categories: 'BICCN_subclass_label', L6 IT.")
```

**Comment by AvolaAmg** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -31135,27 +31141,33 @@ EquivalentClasses(obo:CL_4030062 ObjectIntersectionOf(obo:CL_4023040 ObjectSomeV
...
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292694") Annotation(oboInOwl:hasDbXref "PMID:37609206") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4030065 "An intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 6.")
+AnnotationAssertion(obo:IAO_0000028 obo:CL_4030065 "L6 IT")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292694") Annotation(oboInOwl:hasDbXref "PMID:37609206") Annotation(oboInOwl:hasDbXref "PMID:37824655") Annotation(oboInOwl:hasDbXref "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/") obo:IAO_0000115 obo:CL_4030065 "An transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in L6 of the primary motor cortex. These cells are short untufted pyramidal cells, which could be stellate or inverted.  The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', L6 IT.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:31167]: The first "An" should be "A"

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -29464,7 +29464,7 @@ EquivalentClasses(obo:CL_4023039 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeV
 # Class: obo:CL_4023040 (L2/3-6 intratelencephalic projecting glutamatergic neuron)
 
 AnnotationAssertion(obo:IAO_0000028 obo:CL_4023040 "L2/3-6 IT glut")
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023040 "A intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layers L2/3-6")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34616075") Annotation(oboInOwl:hasDbXref "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/") obo:IAO_0000115 obo:CL_4023040 "A intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layers L2/3-6.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:29467]: There is a grammatical mistake: `A intratelencephalic` &rarr; `An intratelencephalic`

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**